### PR TITLE
(WIP) Adds wp10 form for Persian Wikipedia

### PR DESCRIPTION
**Pull request by halfak**
Bug: https://phabricator.wikimedia.org/T174684

The Persian is incomplete and needs careful review.  @Ladsgroup, do you want to just submit a follow-up commit here?

### CODE REVIEW

File: forms/i18n/wp10/fa.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "@metadata": [],
+    "fa-label": "برگزیده",
+    "ga-label": "خوب",
+    "b-label": "متوسط",
+    "c-label": "ضعیف",
+    "start-label": "ابتدایی",
+    "stub-label": "خرد",
+    "wp10-label": "Assessment",
```
Comment (Ladsgroup) [inline forms/i18n/wp10/fa.json]: The translation of "Assessment" should be "درجه‌بندی"

**Comment by Ladsgroup** [comment]
Beside this one translation thing, I don't see any reason not to merge it.

File: forms/i18n/wp10/fa.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    "more-info-content": "See <a href=\"https://fa.wikipedia.org/wiki/%D9%88%DB%8C%DA%A9%DB%8C%E2%80%8C%D9%BE%D8%AF%DB%8C%D8%A7:%D8%AF%D8%B1%D8%AC%D9%87%E2%80%8C%D8%A8%D9%86%D8%AF%DB%8C\">ویکی‌پدیا:درجه‌بندی</a> for detailed explanations of the assessment classes."
```
Comment (halfak) [inline forms/i18n/wp10/fa.json]: What about this, @Ladsgroup?

Comment (Ladsgroup) [reply forms/i18n/wp10/fa.json]: Thanks for pointing out. That would be:
"برای اطلاعات بیشتر در مورد درجه‌بندی صفحه <a href=\"https://fa.wikipedia.org/wiki/%D9%88%DB%8C%DA%A9%DB%8C%E2%80%8C%D9%BE%D8%AF%DB%8C%D8%A7:%D8%AF%D8%B1%D8%AC%D9%87%E2%80%8C%D8%A8%D9%86%D8%AF%DB%8C\">ویکی‌پدیا:درجه‌بندی</a> را ببینید."

**Comment by halfak** [comment]
DONE. Please re-review. :)

**Pull request merged**